### PR TITLE
[INDY-1432] Added a request type for config ledger.

### PR DESCRIPTION
**Pull request by NataliaDracheva**
Signed-off-by: NataliaDracheva <natalia.dracheva@dsr-corporation.com>

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

**Comment by skhoroshavin** [comment]
(ci) ok to test

File: scripts/performance/perf_load/perf_gen_req_parser.py [hunk 1]
```diff
@@ -14,7 +15,7 @@ class ReqTypeParser:
          "revoc_reg_entry": RGEntryRevoc, "get_nym": RGGetNym, "get_attrib": RGGetAttrib, "get_schema": RGGetSchema,
          "get_cred_def": RGGetDefinition, "get_revoc_reg_def": RGGetDefRevoc, "get_revoc_reg": RGGetEntryRevoc,
          "get_revoc_reg_delta": RGGetRevocRegDelta, "get_payment_sources": RGGetPaymentSources, "payment": RGPayment,
-         "verify_payment": RGVerifyPayment}
+         "verify_payment": RGVerifyPayment, "config": RGConfig}
```
Comment (dsurnin) [inline scripts/performance/perf_load/perf_gen_req_parser.py]: Name "config" is too general. Rename it to something like "pool_cfg_writes" and RGConfig rename accordingly.

File: scripts/performance/perf_load/perf_req_gen_config_ledger.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from indy import ledger
+
+from perf_load.perf_utils import rawToFriendly
+from perf_load.perf_req_gen import RequestGenerator
+
+
+class RGConfig(RequestGenerator):
+    _req_types = ["111"]
+
+    def _rand_data(self):
```
Comment (dsurnin) [inline scripts/performance/perf_load/perf_req_gen_config_ledger.py]: this functions generates random data for the req. For config type it is enough to generate just random int like this
def _rand_data(self):
      return str(random.randint(0, 99999999))

Comment (NataliaDracheva) [reply scripts/performance/perf_load/perf_req_gen_config_ledger.py]: Fixed.

**Pull request closed without merging**